haskell/package.yaml

```diff
@@ -45,6 +45,8 @@ library:
     - bytestring
     - uncaught-exception
     - servant-client
+    - template-haskell
+    - directory
```

Comment:
I was going to mention how `directory` is not in alphabetical order, but oh boy it's not the only one :sweat_smile: 

Would you mind please fixing that (a fixup to any commit that changes this file is ok IMO).

Reply:
Done